libraries/Microsoft.Bot.Builder.Dialogs/Constants/DialogPath.cs

```diff
@@ -1,6 +1,42 @@
-﻿namespace Microsoft.Bot.Builder.Dialogs
+﻿using System.Data;
+
+namespace Microsoft.Bot.Builder.Dialogs
 {
     public class DialogPath
     {
+        /// <summary>
+        /// Counter of emitted events.
+        /// </summary>
+        public const string EventCounter = "dialog.eventCounter";
```

Comment:
>EventCounter [](start = 28, length = 12)

TurnPath constants are all caps. I think we should make them the same.

Reply:
Done

---
In reply to: [360160246](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r360160246) [](ancestors = 360160246)